src/utils/workspaceUtils.ts

```diff
@@ -73,6 +79,17 @@ export async function getWorkspaceFolder(context: IActionContext): Promise<Works
             // don't wait
             void commands.executeCommand('vscode.openFolder', uri[0]);
             context.telemetry.properties.noWorkspaceResult = openExistingProject;
+        } else {
+            const wizardContext: Partial<IStaticWebAppWizardContext> & IActionContext = context;
```

Comment:
It's a little unintuitive, but you should be adding the new steps to the wizard here: https://github.com/microsoft/vscode-azurestaticwebapps/blob/main/src/tree/SubscriptionTreeItem.ts#L53

The way you have it now, it doesn't actually integrate into the wizard properly so you can't leverage the current flow or use the back button.

That way you won't have to change some steps to accept the Partial contexts either.  

Based off of the result of `getWorkspaceFolder`, we can change the creation flow in https://github.com/microsoft/vscode-azurestaticwebapps/blob/main/src/commands/createStaticWebApp/createStaticWebApp.ts to skip all of our validation code, create the SWA based off the template, and then ask to do the clone/reload at the end.